.gradient/symlink_datasets_and_caches.py

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env -S python3 -u
+import json
+import time
+from pathlib import Path
+import subprocess
+import os
+
+
+# read in symlink config file
+with open(f"{Path(__file__).parent.absolute().as_posix()}/symlink_config.json", "r") as f:
+    json_data = f.read()
```

Comment:
I think we can streamline this line a bit: 
```suggestion
json_data = (Path(__file__).resolve().parent / "symlink_config.json").read_text()
```